jobs/build/refresh-images/Jenkinsfile

```diff
@@ -78,12 +87,30 @@ node('openshift-build-1') {
             if ( RHEL != "" ) {
                 rhel_arg = "--rhel ${RHEL}"
             }
-            
+
             sshagent(['openshift-bot']) { // merge-and-build must run with the permissions of openshift-bot to succeed
                 sh "kinit -k -t /home/jenkins/ocp-build.keytab ocp-build/atomic-e2e-jenkins.rhev-ci-vms.eng.rdu2.redhat.com@REDHAT.COM"
+
+                // ose_images method
                 sh "ose_images.sh --user ocp-build update_docker ${rhel_arg} --version ${VERSION_OVERRIDE} --release ${RELEASE_OVERRIDE} --force --branch rhaos-${OSE_MAJOR}.${OSE_MINOR}-rhel-7 --group ${OSE_GROUP}"
                 sh "ose_images.sh --user ocp-build build --branch rhaos-${OSE_MAJOR}.${OSE_MINOR}-rhel-7 --group ${OSE_GROUP} --repo ${OSE_REPO}"
                 sh "sudo env \"PATH=${env.PATH}\" ose_images.sh push --branch rhaos-${OSE_MAJOR}.${OSE_MINOR}-rhel-7 --group ${OSE_GROUP}"
+                // end ose_images method
+
+                // OIT method
+                buildlib.write_sources_file()
+                buildlib.oit """
```

Comment:
Should be no rebase during refresh. Look at what ose_images is doing.

Reply:
Fixed.